docs/concepts/data-collections.md

```diff
@@ -0,0 +1,328 @@
+---
+id: data-storage
+title: Data Storage / Collections
+sidebar_label: Data Storage / Collections
+---
+
+## Overview
+
+> All data stored on the NEAR blockchain is done in key / value pairs. There are several collection methods in the SDKs we've created that will help you store your data on chain.
+>
+> - [`near-sdk-as`](https://github.com/near/near-sdk-as) for [AssemblyScript](https://www.assemblyscript.org/) smart contracts
+> - [`near-sdk-rs`](https://github.com/near/near-sdk-rs) for [Rust](https://www.rust-lang.org/) smart contracts
+
+---
+
+## AssemblyScript Collection Types
+
+| Type                                                                           | Iterable | Clear all values | Preserves order |
```

Comment:
Should mean order of keys.  Which is insertion order in the case of the unordered map and sorted order in terms of the trees.

Reply:
updated :)